src/include/sof/audio/host_copier.h

```diff
@@ -108,5 +109,10 @@ int host_zephyr_params(struct host_data *hd, struct comp_dev *dev,
 int host_zephyr_copy(struct host_data *hd, struct comp_dev *dev, copy_callback_t cb);
 void host_update_position(struct host_data *hd, struct comp_dev *dev, uint32_t bytes);
 void host_one_shot_cb(struct host_data *hd, uint32_t bytes);
+int copier_host_create(struct comp_dev *parent_dev, struct copier_data *cd,
+		       struct comp_ipc_config *config,
+		       const struct ipc4_copier_module_cfg *copier_cfg,
+		       int dir);
+void copier_host_free(struct copier_data *cd);
```

Comment:
Do these need to be in host_copier.h? or simply copier.h?

Reply:
host_copier.h used to stored copier_host.c function prototype, so put it here should be fine